source/items/platearmor.md

```diff
@@ -0,0 +1,27 @@
+---
+title: Plate Armor
+layout: default
+---
+# Plate Armor
+
+<div class="infobox box text-center">
+    <p style="text-align:center;"><img src="../../assets/images/items/plate_armor_helmet.png" alt="Plate Armor Helmet"></p>
+    <p style="text-align:center;"><img src="../../assets/images/items/plate_armor_chest.png" alt="Plate Armor Chestpiece"></p>
+    <p style="text-align:center;"><img src="../../assets/images/items/plate_armor_legs.png" alt="Plate Armor Leggings"></p>
+    <p style="text-align:center;"><img src="../../assets/images/items/plate_armor_boots.png" alt="Plate Armor Boots"></p>
+    Plate Armor can only be crafted by knowledgeable Blacksmiths.
+</div>
+<br>
+
+Plate Armor is a type of armor added by MineColonies. It is moderately durable and offers excellent protection compared to conventional iron armor, but has poor enchantability. Plate Armor is be crafted by the [Blacksmith](../../source/workers/blacksmith), after their [workshop](../../source/buildings/blacksmith) has reached level 4, and the Plate Armor research has been completed at the [University](../../source/buildings/university).
+
+## Material Costs
+
+| Item       | Initial Crafting Requirement    |
+| ---------- | ------------------------------- |
+| Helmet     | 4 iron ingot, 1 leather, 1 coal |
+| Chestplate | 7 iron ingot, 1 leather, 3 coal |
+| Legs       | 6 iron ingot, 1 leather, 4 coal |
+| Boots      | 3 iron ingot, 1 leather, 1 coal |
+
+Like many other recipes, the Blacksmith may improve the recipe efficiency over time, reducing these costs. They will always cost at least one iron, one leather, and one coal.
```

Comment:
maybe "Like most other recipes" to imply that the majority of recipes can be improved